### ISSUE TITLE
[node-server-state-changed] Add 'Continue if' option / filtering by state along with entity_id

### ISSUE THREAD

**Issue by RePeet13** [AYapejian/node-red-contrib-home-assistant#10]
In home assistant, a lot of the time I use the To keyword in automations. It would be really handy to be able to use that in the node-server-state-changed node. Currently I have to work around it by adding a function node after almost every node to filter out states that I don't care about in that flow

**Comment by tehniemer** [comment]
Are you talking about the ```events: all``` node? If so, can you just use the ```events: state``` node for what you're specifically looking for?

**Comment by RePeet13** [comment]
nono, im using the events:state, but what I'm trying to replicate is the 'to state' part of automations. Like I only need to do something if one state on one entity happens. Yes i could do this with a second node like formula or switch, but it'd be nice to have it all in one step

**Comment by mszoernyi** [comment]
An example for this:

**events:state node**
```
Name: below horizon?
Server: Homeassistant
Entity ID: `sun.sun`
Halt if:
Continue If: below_horizon
```

This is one of the use cases I have a lot. Right now you have to have a switch or function filtering for everything but the state you are looking for.

**Comment by AYapejian** [comment]
Good idea, I'll mark as enhancement and address after some of the other bugs that have been reported here lately.

**Comment by oakbrad** [comment]
I agree this would be great! Sort of the opposite of 'halt if state'

**Comment by AYapejian** [comment]
I think this should all be handled now in the new trigger state node I just released with 0.3.0. Constraints and custom output options should cover the halt / continue if scenarios
